test/http.ts

```diff
@@ -309,19 +309,23 @@ test('DNS IPv4', withServer, async (t, server, got) => {
 	t.true(isIPv4(response.body));
 });
 
-if (IPv6supported) {
-	test('DNS IPv6', withServer, async (t, server, got) => {
-		server.get('/ok', echoIp);
+// Travis CI Ubuntu Focal VM does not resolve IPv6 hostnames
+testIPv6('DNS IPv6', withServer, async (t, server, got) => {
+	server.get('/ok', echoIp);
 
+	try {
 		const response = await got('ok', {
 			dnsLookupIpVersion: 'ipv6'
 		});
 
 		t.true(isIPv6(response.body));
-	});
-}
+	} catch (error) {
+		// If network unreachable (happens on TravisCI with Ubuntu Bionic and Focal)
+		t.is(error.code, 'ENETUNREACH');
+	}
```

Comment:
I guess this is moot now?

Reply:
This is still valid because we are testing on Focal (20.04) on Travis CI.
When we switch to Gitlab CI we will need to understand how they treat the IPv6 addresses.

Reply:
> When we switch to Gitlab CI

Do you mean GitHub?

Reply:
> > When we switch to Gitlab CI
> 
> Do you mean GitHub?

Yeah, sorry, I meant Github Actions